### ISSUE TITLE
Doesn't work when a line has multiple words

### ISSUE THREAD

**Issue by cvank** [PK-Yadav/java-learning-pappuYadav#15]
https://github.com/PK-Yadav/java-learning-pappuYadav/blob/dd79e089f6df3b8fa70b39388871944d9a911299/multi-threading-task2/repeating-word-count/src/main/java/com/java/learning/task/multithreading/tasks/wordcount/WordCounterService.java#L52

**Comment by PK-Yadav** [comment]
yes, actually i am assuming line will have only one word, in other case like lines can contain words with space separated or comma separated then i have to use the parser, Like DocumentReaderWritter parse which i have written in previous module.
